view/mongo/src/main/kotlin/io/holunda/polyflow/view/mongo/MongoViewService.kt

```diff
@@ -435,6 +435,20 @@ class MongoViewService(
     deleteDataEntry(dataIdentityString(entryType = event.entryType, entryId = event.entryId))
   }
 
+  /**
+   * Delivers data entry anonymized event.
+   */
+  @Suppress("unused")
+  @EventHandler
+  fun on(event: DataEntryAnonymizedEvent, metaData: MetaData) {
+    logger.debug { "Business data entry anonymized $event" }
+    dataEntryRepository.findNotDeletedById(dataIdentityString(entryType = event.entryType, entryId = event.entryId))
+      .map { oldEntry -> event.toDocument(oldEntry) }.map { dataEntryRepository.save(it) }
+      .then(updateDataEntryQuery(QueryDataIdentity(entryType = event.entryType, entryId = event.entryId)))
+      .block()
+    throw NotImplementedError("Anonymizing data entries is not yet supported in the mongo view")
```

Comment:
Why are you throwing it here? The implementation looks reasonable already...

Reply:
oops, added it later and forgot to remove the NotImplementedError ^^